mailsender/tool/src/webapp/WEB-INF/bundle/Messages.properties

```diff
@@ -58,6 +58,7 @@ exception.generic=A problem has occurred: {0}
 error.unknown.role.type=An unknown role type was requested.
 error.no.recipients=No recipients specified.
 error.no.valid.recipients=No valid recipients specified.
+error.attachmentsize.exception=You added too many attachments. Limit: {0} MB
```

Comment:
Is the number of attachments what causes the "-1"?

"too many attachments" and a size don't seem to be compatible units.

Reply:
No, it's just the way the java library reports size. -1 is to be expected, unless you dig in deeper and get the underlying file size, which is what I've done.

Reply:
Ah, I get you. The message is a bit odd.